pkg/api/handlers/compat/containers.go

```diff
@@ -466,7 +466,7 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 	ports := make(nat.PortSet)
 	for p := range inspect.HostConfig.PortBindings {
 		splitp := strings.Split(p, "/")
-		port, err := nat.NewPort(splitp[0], splitp[1])
+		port, err := nat.NewPort(splitp[1], splitp[0])
```

Comment:
This looks like we should have an error check here as well, to make sure that the splitp is at least two items.